string sync_mode
uint8 index
---
string response